pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -0,0 +1,50 @@
+{
+  stdenv, lib, fetchFromGitHub, makeDesktopItem, prusa-slicer
+}:
+let
+  appname = "SuperSlicer";
+  version = "2.2.53.1";
+  pname = "super-slicer";
+  description = "PrusaSlicer fork with more features and faster development cycle";
+  override = super: {
+    inherit version pname;
+
+    src = fetchFromGitHub {
+      owner = "supermerill";
+      repo = "SuperSlicer";
+      sha256 = "sha256-CAhwmQ63N/XJYToTnIV84lNnjDGNbkmYPzNKNL/wVxs=";
```

Comment:
Just out of interest, what do you use to generate the hashes? I get them in the form of e.g.

```
06spy2zk8jik7yc4jvld6666flz2gj2rq4rsc74zadxp1scp0208
```

not encoded as base56 like above.

Reply:
I did nix-prefetch-github, and on nix-unstable it always returns this type of hash

Reply:
@cab404 please, can you run it with `sha256= stdenv.lib.fakeSha256;`?

Reply:
@AndersonTorres Why?

Reply:
@dali99 well, mainly by curiousity. This is the way I use to modify the source hashes: the Nix command fails and then spits out the true hash.

Reply:
> @dali99 well, mainly by curiousity. This is the way I use to modify the source hashes: the Nix command fails and then spits out the true hash.

no problemo
but several packages already using that form of hash in nixpkgs

Reply:
> @cab404 please, can you run it with `sha256= stdenv.lib.fakeSha256;`?

```
error: --- Error --- nix-daemon
hash mismatch in fixed-output derivation '/nix/store/nrgz6mb2zjk4lhk9qmqj32816qgragpf-source':
  wanted: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
  got:    sha256-CAhwmQ63N/XJYToTnIV84lNnjDGNbkmYPzNKNL/wVxs=
error: --- Error ------------------------------------------------------------ nix-build
error: --- Error --- nix-daemon
1 dependencies of derivation '/nix/store/1k3hzlqcgprfg4a095hsbfwjly4scyld-super-slicer-2.2.53.1.drv' failed to build
```

Reply:
OK to me, hehe!